common-rest/src/test/java/org/uniprot/api/common/repository/stream/rdf/RDFStreamerTest.java

```diff
@@ -191,4 +93,4 @@ void idsToRDFTupleStoreStream() {
                         + "</rdf:RDF>",
                 rdfString);
     }
```

Comment:
add tests for multiple ids and empty response

Reply:
done